apps/prairielearn/src/pages/authLogin/authLogin.html.ts

```diff
@@ -311,42 +327,54 @@ export function AuthLoginUnsupportedProvider({
       ${run(() => {
         if (showUnsupportedMessage) {
           return html`
-            <div class="alert alert-danger text-center my-4" role="alert">
-              The authentication provider you tried to use is not supported by your institution.
-              ${supportsNonLti ? 'Please use a supported provider.' : ''}
-              ${!supportsNonLti && supportsLti
-                ? "You must start a session from your course's Learning Management System (LMS)."
-                : ''}
-              ${supportedProviders.length === 0
-                ? 'Contact your institution for more information.'
-                : ''}
+            <div
+              class="${clsx('alert alert-danger text-center', {
+                'mb-0': loginOptions.length === 0,
+              })}"
+              role="alert"
+            >
+              The authentication method you tried to use is not supported by your institution.
```

Comment:
I switched from "provider" to "method" in user-facing copy; I felt that was just slightly friendlier. "Provider" is really just what we call these things internally.